pyproject.toml

```diff
@@ -31,7 +31,6 @@ requires-python = ">=3.10,<3.13"
 dependencies = [
      "pymatgen>=2024.10.22",
      "numpy<3.0.0",
-     "typing",
```

Comment:
I guess we don't need to install `typing` now, it's [built in from 3.5](https://docs.python.org/3/library/typing.html):
> Added in version 3.5.